graphql/response.go

```diff
@@ -14,6 +15,9 @@ import (
 type Response struct {
 	Errors     gqlerror.List          `json:"errors,omitempty"`
 	Data       json.RawMessage        `json:"data"`
+	Label      string                 `json:"label,omitempty"`
+	Path       ast.Path               `json:"path,omitempty"`
+	HasNext    bool                   `json:"hasNext"`
```

Comment:
What do you think about adding omitempty here? @fiatjaf @UnAfraid

Reply:
Looks good to me.